ranch/ranch.go

```diff
@@ -174,21 +176,7 @@ func (r *Ranch) Acquire(rType, state, dest, owner, requestID string) (*crds.Reso
 		}
```

Comment:
you could call the new getter right before you delete the request here (I can't link the exact line, but just go up a few lines)

Reply:
Working on it now.

Reply:
Updated.